docs/react/3.optimization/error-boundaries.md

```diff
@@ -3,24 +3,27 @@ title: Error Boundaries
 sidebar_label: Error Boundaries
 ---
 
-import Img from '../../../src/components/Img';
+import Img from '../../../src/components/Img';import Hint from '../../../src/components/Hint';
 
-In the past, JavaScript errors inside components used to corrupt React’s internal state and cause it to emit cryptic errors on next renders. These errors were always caused by an earlier error in the application code, but React did not provide a way to handle them gracefully in components, and could not recover from them.
+In the past, JavaScript errors inside components used to corrupt React’s internal state and cause it to emit cryptic errors on next renders. These errors were always caused by an earlier error in the application code, but React did not provide a way to handle them gracefully in components, and could not recover from them. Like this in your console: <Img w="700" src='https://cosmos-x.oss-cn-hangzhou.aliyuncs.com/UMQoLL.png'/>
```

Comment:
Note that the app without Error Boundaries will be an empty page when error occurs. Better to shown this in the screenshot.